tags/tag_set.go

```diff
@@ -43,17 +41,19 @@ type TagSet struct {
 	m map[Key][]byte
```

Comment:
(on line 39 but GH won't let me add a comment since it isn't in the diff)

Remove `TagSetBuilder` from doc comment

Reply:
Done